extensions/sql-migration/src/constants/strings.ts

```diff
@@ -1289,3 +1290,113 @@ export const SQLDB_COL_COPY_DURATION = localize('sql.migration.sqldb.column.copy
+
+export const TDE_WIZARD_ADS_CERTS_INFO = localize('sql.migration.network.share.header.text', "Please enter a location where the SQL server will export the certificates. Please ensure the SQL server service has access access to this path. The current user should have administrator privileges on the computer where this network path is.");
+
+export const TDE_WIZARD_CERTS_NETWORK_SHARE_LABEL = localize('sql.migration.tde.wizard.network.share.label', "Network path for certificate");
+export const TDE_WIZARD_CERTS_NETWORK_SHARE_PLACEHOLDER = localize('sql.migration.tde.wizard.network.share.placeholder', "Enter network path");
+export const TDE_WIZARD_CERTS_NETWORK_SHARE_INFO = localize('sql.migration.tde.wizard.network.share.info', "Network path where certificate will be placed.");
+
+export const TDE_WIZARD_CERTS_DOMAIN_LABEL = localize('sql.migration.tde.wizard.domain.label', "Domain");
+export const TDE_WIZARD_CERTS_DOMAIN_PLACEHOLDER = localize('sql.migration.tde.wizard.domain.placeholder', "Enter Domain");
+export const TDE_WIZARD_CERTS_DOMAIN_INFO = localize('sql.migration.tde.wizard.domain.info', "Domain for the credentials provided.");
+
+export const TDE_WIZARD_CERTS_USERNAME_LABEL = localize('sql.migration.tde.wizard.username.label', "Username");
+export const TDE_WIZARD_CERTS_USERNAME_PLACEHOLDER = localize('sql.migration.tde.wizard.username.placeholder', "Enter Username.");
+export const TDE_WIZARD_CERTS_USERNAME_INFO = localize('sql.migration.tde.wizard.username.info', "Username for the credentials provided.");
+
+export const TDE_WIZARD_CERTS_PASSWORD_LABEL = localize('sql.migration.tde.wizard.password.label', "Password");
+export const TDE_WIZARD_CERTS_PASSWORD_PLACEHOLDER = localize('sql.migration.tde.wizard.password.placeholder', "Enter password.");
+export const TDE_WIZARD_CERTS_PASSWORD_INFO = localize('sql.migration.tde.wizard.password.info', "Password for the credentials provided.");
+
+export const TDE_MIGRATE_BUTTON = localize('sql.migration.tde.button.migrate', "Migrate TDE certificates");
+
+
+export const TDE_MIGRATE_STATE_CANCELED = localize('sql.migration.tde.migrate.state.canceled', "Canceled");
+export const TDE_MIGRATE_STATE_PENDING = localize('sql.migration.tde.migrate.state.pending', "Pending");
+export const TDE_MIGRATE_STATE_RUNNING = localize('sql.migration.tde.migrate.state.running', "Running");
+export const TDE_MIGRATE_STATE_SUCCEEDED = localize('sql.migration.tde.migrate.state.succeeded', "Succeeded");
+export const TDE_MIGRATE_STATE_FAILED = localize('sql.migration.tde.migrate.state.failed', "Failed");
+
+export const TDE_MIGRATEDIALOG_TITLE = localize('sql.migration.validation.dialog.title', "Certificates Migration");
+export const TDE_MIGRATE_DONE_BUTTON = localize('sql.migration.tde.migrate.done.button', "Done");
+export const TDE_MIGRATE_HEADING = localize('sql.migration.tde.migrate.heading', "Migrating the certificates following databases:");
```

Comment:
nit: "certificates **from** the following databases"

Reply:
Fixed